packages/babel-plugin-htm/index.mjs

```diff
@@ -1,4 +1,4 @@
-import htm from 'htm';
+import { build,	treeify } from '../../src/build.mjs';
```

Comment:
Does this get bundled? If not the path will fail when published.

Reply:
Yes, `build` and `treeify` get bundled to `dist/babel-plugin-htm.js` (and the corresponding `.mjs` file).

(Fixed the weird tab between `{ build,` and `treeify }` though 🙂)